website/docs/components/table/advanced-table/partials/content/content.md

```diff
@@ -0,0 +1 @@
+For general content recommendations, refer to our [Tabular guidelines](/content/tabular-guidelines) documentation.
```

Comment:
I would change this to "Table content guidelines" URL: /content/table-content-guidelines to be clearer for consumers.

Or, if you want to keep the work "tabular" in this context, I would change the text to [Tabular data guidelines](/content/tabular-data-guidelines). This is still less clear, I think, in this context but more accurate. "Tabular data" describes data presented in a tabular format. If you just say, "Tabular guidelines", it doesn't quite make sense to me.

Reply:
You will need to modify the page URL with the above suggestion